src/ray/core_worker/object_interface.h

```diff
@@ -0,0 +1,60 @@
+#ifndef RAY_CORE_WORKER_OBJECT_INTERFACE_H
+#define RAY_CORE_WORKER_OBJECT_INTERFACE_H
+
+#include "common.h"
+#include "ray/common/io.h"
+#include "ray/id.h"
+#include "ray/status.h"
+
+namespace ray {
+
+class CoreWorker;
+
+/// The interface that contains all `CoreWorker` methods that are related to object store.
+class CoreWorkerObjectInterface {
+ public:
+  CoreWorkerObjectInterface(CoreWorker *core_worker) { core_worker_ = core_worker; };
+
+  /// Put an object into object store.
+  ///
+  /// \param[in] buffer Data buffer of the object.
```

Comment:
We haven't been using `[in]` anywhere else, so maybe we should get rid of that.

Reply:
Ok, we discussed in person and it seems like we should keep it and start using `[in]` everywhere.